src/main/resources/fabric.mod.json

```diff
@@ -4,7 +4,7 @@
   "version": "${version}",
 
   "name": "Dynamic Crosshair",
-  "description": "Dynamic Crosshair that changes shape based on what is targeted",
+  "description": "Contextual crosshair style",
```

Comment:
Do you know where this string is used? Is it worth to shorten it here?

Reply:
Not quite sure, maybe it's sort of system mod description

Reply:
Maybe that string uses if "modmenu.summaryTranslation.dynamiccrosshair" key not found in localization file (or there's no localization file for this language)

Reply:
From my quick testing, that string is indeed used if there is no `summaryTranslation` string. However, once that `summaryTranslation` string exists in `en_us.json`, other languages appear to automatically pick that one up.

So I believe we can keep this more verbose (in its old form).

Reply:
Ok, one sec